earth_enterprise/src/installer/install_server.sh

```diff
@@ -27,6 +27,7 @@ SCRIPTDIR=`dirname $0`
 PUBLISHER_ROOT="/gevol/published_dbs"
 INSTALL_LOG="$INSTALL_LOG_DIR/geserver_install_$(date +%Y_%m_%d.%H%M%S).log"
 BACKUP_DIR="$BASEINSTALLDIR_VAR/server-backups/$(date +%Y_%m_%d.%H%M%S)"
+BACKUP_DATA_DIR=$BACKUP_DIR/data.backup
```

Comment:
On Ubuntu 16.04, at the verification step 12: Do the install and verification once more to verify that the install upgrade also works with PostgreSQL 9.6 installed.

I published a new 3d database from existing published one but enabling only 'POI search'. Then I did run a POI Search by entering "880" on a 3d database with San Francisco roads (as I did in previous steps) to verify that the POI search is still working and got 'Google Earth Client' to crash three times. Not sure if this has any thing to do the databases re-upgrade (if any).

Reply:
I was able to reproduce this issue few times without doing the upgrade. So all points to a bug within Google Earth Enterprise Client'. This does not appear to be related to this PR.